esmvalcore/cmor/_fixes/access/access_esm.py

```diff
@@ -0,0 +1,128 @@
+"""On-the-fly CMORizer for ACCESS-ESM.
+
+Note
+----
+This is the first version of ACCESS-ESM CMORizer in for ESMValCore
+Currently, only two variables (`tas`,`pr`) is fully supported.
+"""
+import logging
+import os
+
+from iris.cube import CubeList
+
+from esmvalcore.cmor._fixes.native_datasets import NativeDatasetFix
+
+logger = logging.getLogger(__name__)
+
+
+class Tas(NativeDatasetFix):
+    """Fix variable(tas) only."""
+
+    def __init__(self, vardef, extra_facets, session, frequency):
+        """Initialise some class variable Heritage from native_dataset."""
+        super().__init__(vardef, extra_facets, session, frequency)
+
+        self.cube = None
+
+        self.current_dir = os.path.dirname(__file__)
+
+    def fix_height_name(self):
+        """Fix variable name of coordinate 'height'."""
+        if self.cube.coord('height').var_name != 'height':
+            self.cube.coord('height').var_name = 'height'
```

Comment:
This is a variable specific function that can go into `Tas`. Note: we also have a function [`NativeDatasetFix.fix_height_metadata`](https://github.com/ESMValGroup/ESMValCore/blob/ee16a958d916f1620a2aa25191e4cc34b3886006/esmvalcore/cmor/_fixes/native_datasets.py#L270-L298) that does that job.